app/models/email.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const emailSchema = new Schema({
+  email: {
+    type: String,
```

Comment:
`unique: true,` will add an index on this column (afaik. about mongodb), making sure we don't end up with duplicates. Then we don't have to explicitly check if the user is already present (and such operation has to be in a transaction as well I guess). :smile: